lookbook/docs/patterns/17-settings-pages.md.erb

```diff
@@ -1 +1,29 @@
-When working on Settings pages, please refer to the Forms pattern for information on how to implement form elements.
\ No newline at end of file
+Settings pages usually have content space that stretches to the full width of the viewport, as there is no other element
+horizontally positioned, except the left side menu.
+
+## Settings page base component
+
+According to the [form width specification](./forms), we avoid forms to stretch without limit. Because of the default
+behaviour of form elements to use the full width of the container, we use a base component that embeds the form inside
+such a wrapping container.
+
+Any view component rendering a form should inherit from `AdminSettingsFormComponent`. The template stays unchanged. This
+way, the rendered form is wrapped into a container with `480px` max-width.
```

Comment:
```suggestion
way, the rendered form is wrapped into a container with `680px` max-width.
```

Reply:
FYI: Made this change my (later) commit.